tests/integrations/client/http_client_test.go

```diff
@@ -384,3 +384,63 @@ func (suite *httpClientTestSuite) TestScheduleConfig() {
 	re.Equal(float64(8), config["leader-schedule-limit"])
 	re.Equal(float64(2048), config["region-schedule-limit"])
 }
+
+func (suite *httpClientTestSuite) TestSchedulers() {
+	re := suite.Require()
+	schedulers, err := suite.client.GetSchedulers(suite.ctx)
+	re.NoError(err)
+	re.Len(schedulers, 0)
```

Comment:
Why is it zero? Shouldn't it be equal to the default number of schedulers?

Reply:
Because the test doesn't wait for the coordinator to start

Reply:
Maybe we don't need to wait for the coordinator to start working, just focus on the functionality of HTTP client